db/db_impl/db_impl_files.cc

```diff
@@ -360,6 +360,13 @@ void DBImpl::DeleteObsoleteFileImpl(int job_id, const std::string& fname,
         &event_logger_, job_id, number, fname, file_deletion_status, GetName(),
         immutable_db_options_.listeners);
   }
+#ifndef ROCKSDB_LITE
+  if (type == kBlobFile) {
+    EventHelpers::NotifyBlobFileDeletion(immutable_db_options_.listeners,
+                                         job_id, number, fname,
+                                         file_deletion_status, GetName());
+  }
```

Comment:
For SST files, the Notify also logs something to the Logger (independent or LITE mode). 

 Is that not a requirement/useful for Blob files?

Reply:
I wasn't sure about it so I didn't add it at first. But @ltamasi  let me know if we want to log in  the Logger or not. I will make the changes accordingly.